src/VIPServer.java

```diff
@@ -107,24 +107,6 @@ public void main(String[] args) throws HostFailureException, HostNotFoundExcepti
 					// SHOULD BE REPLACED BY
 					// job.carryOn();
 				} else {
-					if (!timer) {
```

Comment:
So if in VIP we send back the number of computing particles to the server, we have to do it in simulation too. 
For the last point the idea is to stop injecting the execution time at some point, isn't it? So my question is: in VIP how is distributed the initial number of particles among the workers? we have to do the same way, even if we do not use it for now. 

My point is that it's better to be as close as possible to the internal logic of VIP from the beginning than having to patch up things later because suddenly there is a "but is not like that in real life".